examples/preview-panel-in-modal/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@algolia/autocomplete-example-preview-panel-in-modal",
+  "description": "Autocomplete example with a preview panel in a modal",
+  "version": "1.0.0-alpha.45",
+  "private": true,
+  "license": "MIT",
+  "main": "index.html",
```

Comment:
We need to remove this field with Parcel 2. See #539.

Reply:
Fixed in 750f1d88.